types/rsvp/rsvp-tests.ts

```diff
@@ -118,7 +118,29 @@ function testHash() {
 }
 
 function testHashSettled() {
-    // TODO: add test
+    function isFulfilled<T>(state: RSVP.PromiseState<T>): state is RSVP.Resolved<T> {
```

Comment:
Oh, this is nice; I'm going to steal it and use it over in the `allSettled` in short order.